pytest.ini

```diff
@@ -9,4 +9,6 @@ filterwarnings =
     # Remove after support for Python 3.7 is dropped
     ignore:After January 1, 2024, new releases of this library will drop support for Python 3.7:DeprecationWarning
     # Remove warning once https://github.com/googleapis/python-pubsub/issues/1067 is fixed
-    ignore:The return_immediately flag is deprecated and should be set to False.:DeprecationWarning
\ No newline at end of file
+    ignore:The return_immediately flag is deprecated and should be set to False.:DeprecationWarning
+    # Remove warning once https://github.com/googleapis/python-pubsub/issues/1073 is fixed
+    ignore:get_mtls_endpoint_and_cert_source is deprecated.:DeprecationWarning
```

Comment:
I transferred both issues to https://github.com/googleapis/gapic-generator-python because the filter can only be removed once the generated tests are fixed. (The fix in gapic-generator-python should be that deprecation warnings don't bubble up when we're testing things that are known to be deprecated.)
```
    # Remove warning once https://github.com/googleapis/gapic-generator-python/issues/1938 is fixed
    ignore:The return_immediately flag is deprecated and should be set to False.:DeprecationWarning
    # Remove warning once https://github.com/googleapis/gapic-generator-python/issues/1939 is fixed
    ignore:get_mtls_endpoint_and_cert_source is deprecated.:DeprecationWarning
```

Reply:
Please could you update the links to point to issues in https://github.com/googleapis/gapic-generator-python instead of https://github.com/googleapis/python-pubsub